src/app/controller/Login-controller.js

```diff
@@ -0,0 +1,15 @@
+import VerifyLoginDetails from '../model/Login-model';
+
+export default async function CheckLoginDetails(username,password){
+    try {
+        const result = await VerifyLoginDetails(username,password);
+        if (result === "error"){
+            return("error");
+        } else{
+            console.log(`in the controller, the value is ${result.message}`)
+            return result;
+        }
+    } catch(error){
+        return "error";
```

Comment:
We must correctly handle errors. Take a look at this link I've found in Google https://kinsta.com/blog/errors-in-javascript/